osmosis-1/osmosis-1.chainlist.json

```diff
@@ -4986,8 +4986,8 @@
       "chain_name": "medasdigital",
       "status": "live",
       "network_type": "mainnet",
-      "pretty_name": "Medas Digital Network",
```

Comment:
Please don't update this chainlist file directly

Reply:
Ok, sorry seeing it, only osmosis.zone_assets.json.. The Pool info is now automatically pulled? Last time I had to edit some file and enter the initial osmo pool..

Reply:
Ok, should I do a new PR without the chainlist or can you ignore the chainlist for the merge?

Reply:
Just undo the changes to that file in your branch. We can still keep this PR

Reply:
OK, thx.. I reverted the changes.. so for osmosis only the change of the ibc channel and the features seems to be required..